eclair-core/src/main/scala/fr/acinq/eclair/db/PaymentsDb.scala

```diff
@@ -26,7 +26,9 @@ import fr.acinq.eclair.{MilliSatoshi, ShortChannelId}
 
 import scala.compat.Platform
 
-trait PaymentsDb extends IncomingPaymentsDb with OutgoingPaymentsDb
+trait PaymentsDb extends IncomingPaymentsDb with OutgoingPaymentsDb {
+  def listPaymentsOverview(limit: Int): Seq[PlainPayment]
```

Comment:
Why not put that in its own trait? 
```scala
trait PaymentsDb extends IncomingPaymentsDb with OutgoingPaymentsDb with PaymentOverviewDb

trait PaymentOverviewDb {
    def listPaymentsOverview(limit: Int): Seq[PlainPayment]
}
```

This way other components in eclair-core don't need to care about this payment overview and use either the `IncomingPaymentsDb` or `OutgoingPaymentsDb` trait, whereas mobile apps can leverage this new trait.